steam/state.py

```diff
@@ -2387,7 +2389,15 @@ async def handle_notifications(self, msg: notifications.GetSteamNotificationsRes
                             log.info("Unknown commentable type %d", type)
                             continue
                     try:
-                        self.dispatch("comment", await commentable.fetch_comment(int(body["cgid"])))
+                        comment = await commentable.fetch_comment(int(body["cgid"]))
+                        async with self._process_comment_lock:  # prevents multiple dispatch for a single comment
+                            if comment.id in self._processed_comment_ids:
+                                log.debug("Ignoring processed comment")
```

Comment:
```suggestion
                                log.debug("Ignoring processed comment: %s", comment.id)
```
Might be helpful to display them if they're having dupes

Reply:
Got it! Added your suggestion in the code.